apps/meteor/client/views/room/modals/FileUploadModal/FileUploadModal.tsx

```diff
@@ -9,8 +9,10 @@ import FilePreview from './FilePreview';
 
 type FileUploadModalProps = {
 	onClose: () => void;
+	queue?: File[];
```

Comment:
The changes in the Modal are not necessary anymore, right? Since the multi file upload will be handled inside the composer. Only the options to send audio and video are still openening the modal, which are single items, and we can move to be handled in the composer later as well.

Reply:
done sir changed uploadFiles and FileUploadModal to handle single files for audio and video sharing